test/account/deposit.ts

```diff
@@ -1,23 +1,36 @@
 import * as helpers from '../helpers/contract-helpers';
 
-let ssvNetworkContract: any;
+import { expect } from 'chai';
+import { trackGas, GasGroup } from '../helpers/gas-usage';
+
+let ssvNetworkContract: any, clusterResult1: any, minDepositAmount: any;
 
 describe('Deposit Tests', () => {
   beforeEach(async () => {
-    const contractData = await helpers.initializeContract();
-    ssvNetworkContract = contractData.contract;
-  });
+    // Initialize contract
+    ssvNetworkContract = (await helpers.initializeContract()).contract;
+
+    // Register operators
+    await helpers.registerOperators(0, 12, helpers.CONFIG.minimalOperatorFee);
 
-  it('Deposit', async () => {
+    minDepositAmount = (helpers.CONFIG.minimalBlocksBeforeLiquidation + 10) * helpers.CONFIG.minimalOperatorFee * 4;
```

Comment:
what do you mean why? instead of to use magic unclear numbers in the test - we know that minDepositAmount covers  threshold period before liquidation period for all validator operators list